app/src/main/AndroidManifest.xml

```diff
@@ -41,6 +41,7 @@
         android:configChanges="locale|orientation|screenSize"
         android:description="@string/app_description"
         android:hasFragileUserData="false"
+        android:usesCleartextTraffic="true"
```

Comment:
can you please explain why this is necessary, my gut reaction is to be adverse to this

Reply:
😆 I would have had the same reaction. Guess, this needs a comment.
It is needed, because the UPnP IGD protocol is unencrypted HTTP.

Would be happy, if we could limit that to local networks, but I doubt this is possible.

Reply:
Actually, I think it is. I can find some documentation when I get on my computer later, but I've seen this sort of thing before.

Reply:
It's not about talking to localhost (same device), it's about talking to some router on the local network, which could potentially be on *any* IP address, but typically is on a private IP address only. https://en.wikipedia.org/wiki/Private_network